catalog/transformer/catalog-transformer-mgmpv2/src/main/java/org/codice/alliance/catalog/mgmp/converter/MgmpConverter.java

```diff
@@ -0,0 +1,1351 @@
+
+        if (starts.size() == ends.size()) {
+            for (int i = 0; i < starts.size(); i++) {
+
+                String start = starts.get(i);
+                String end = ends.get(i);
+
+                int elementIndex = i + 1;
+
+                if (start.equals(end)) {
+                    pathValueTracker.add(new Path(Utilities.replaceIndex(
+                            GMD_TEMPORAL_TIME_INSTANT_ID_PATH,
+                            elementIndex)), "GMLID_c1dcde60-9aaf-11e5-a565-0002a5d5c51b");
+                    pathValueTracker.add(new Path(Utilities.replaceIndex(GMD_TEMPORAL_INSTANT_PATH,
+                            elementIndex)), start);
+                } else {
+                    pathValueTracker.add(new Path(Utilities.replaceIndex(
+                            GMD_TEMPORAL_TIME_PERIOD_ID_PATH,
+                            elementIndex)), "GMLID_b3bc8920-9aaf-11e5-967d-0002a5d5c51b");
+                    pathValueTracker.add(new Path(Utilities.replaceIndex(GMD_TEMPORAL_START_PATH,
+                            elementIndex)), start);
+                    pathValueTracker.add(new Path(Utilities.replaceIndex(GMD_TEMPORAL_END_PATH,
+                            elementIndex)), end);
+                }
+            }
+        }
+    }
+
+    private void addVerticalElement(XstreamPathValueTracker pathValueTracker,
+            MetacardImpl metacard) {
+
+        final AtomicBoolean addVerticalCRS = new AtomicBoolean(false);
+
+        Utilities.addFieldIfString(pathValueTracker, metacard, metacard1 -> {
+            OptionalDouble optionalDouble = Utilities.getSerializables(metacard1, Location.ALTITUDE)
+                    .stream()
+                    .filter(Double.class::isInstance)
+                    .mapToDouble(Double.class::cast)
+                    .min();
+            if (optionalDouble.isPresent()) {
+                addVerticalCRS.set(true);
+                return Optional.of(Double.toString(optionalDouble.getAsDouble()));
+            }
+            return Optional.empty();
+        }, GMD_MIN_ALTITUDE_PATH);
+
+        Utilities.addFieldIfString(pathValueTracker, metacard, metacard1 -> {
+            OptionalDouble optionalDouble = Utilities.getSerializables(metacard1, Location.ALTITUDE)
+                    .stream()
+                    .filter(Double.class::isInstance)
+                    .mapToDouble(Double.class::cast)
+                    .max();
+            if (optionalDouble.isPresent()) {
+                addVerticalCRS.set(true);
+                return Optional.of(Double.toString(optionalDouble.getAsDouble()));
+            }
+            return Optional.empty();
+        }, GMD_MAX_ALTITUDE_PATH);
+
+        if (addVerticalCRS.get()) {
+            pathValueTracker.add(new Path(GMD_VERTICAL_CRS_XLINK_HREF_PATH),
+                    "http://ww.opengis.net/def/crs/EPSG/0/5701");
+            pathValueTracker.add(new Path(GMD_VERTICAL_CRS_XLINK_TITLE_PATH), "Newlyn Height");
```

Comment:
That's the name of the coordinate ref system used for height, as i understand it. Do you think it should be something different?